### ISSUE TITLE
No USB serial ports showing up for board

### ISSUE THREAD

**Issue by benjaminbradley** [Blinkinlabs/EightByEight#3]
I upgraded my Mac OS and then it could no longer see the 8x8 board on the Arduino app's Ports menu when I plugged it in.
It turns out that:

> Yosemite 10.9 (and OS X 10.10 and newer) requires a signed USB driver kext (kernel extension). You can find one on the FTDI web site here: http://www.ftdichip.com/Drivers/VCP.htm
> Use version 2.3 [or the latest version-Ed] for 64-bit OS X 10.9 or later. Install and reboot.
ref: https://arduino.stackexchange.com/a/13825

Adding here in case any other 8x8 users run into this issue as well.

**Comment by cibomahto** [comment]
Interesting- the EightByEight uses an ARM part as a USB to serial converter, so it shouldn't have anything to do with the FTDI driver.

**Comment by benjaminbradley** [comment]
Hmm my usbserialmodem has disappeared again, so it seems the FTDI driver may have been unrelated. Thinking that it was the reboot that fixed it before, I tried that again, but it's still not showing up. LMK if you have any tips! I'm running:
* Mac OS 10.15.1
* Arduino 1.8.13
* ESP8266 Boards 2.5.0 (downgraded based on https://forum.arduino.cc/index.php?topic=622116.msg4414063#msg4414063 )

I'm using the same USB cable that was working a week ago, and it is getting power. I've tried both USB ports available.
